src/main/java/com/thealgorithms/maths/Mean.java

```diff
@@ -0,0 +1,54 @@
+package com.thealgorithms.maths;
+
+/**
+ * https://en.wikipedia.org/wiki/Mean
+ * <p>
+ * by: Punit Patel
+ */
+public class Mean {
```

Comment:
```suggestion
public final class Mean {
    private Mean() {
    }
```
This makes this class a _proper utility class_.

Reply:
Sure makes sense.